sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -131,6 +145,12 @@
      */
     private AnalyticsListener mAnalyticsListener;
 
+    /**
+     * Transmission interval.
+     * Valid value ranges from 3 seconds to 1 day.
```

Comment:
I don't think we need this in the doc for the variable. This is business logic so it should be enough to say it in the method doc.
```suggestion
```